app/jobs/crons/mark_statements_as_payable_job.rb

```diff
@@ -0,0 +1,14 @@
+class Crons::MarkStatementsAsPayableJob < CronJob
```

Comment:
Just a heads up; the cron jobs aren't scheduled automatically as part of the CI in NPQ reg so you'll need to manually run rake task to do this manually in the first instance